lisa/features/gpu.py

```diff
@@ -273,8 +291,7 @@ def _install_cuda_driver(self, version: str) -> None:
                 # dmesg
                 # NVRM: GPU 0001:00:00.0: RmInitAdapter failed! (0x63:0x55:2344)
                 # NVRM: GPU 0001:00:00.0: rm_init_adapter failed, device minor number 0
-                #  switch to use 495
-                self._node.os.install_packages("cuda-drivers-495")
+                self._node.os.install_packages("cuda-drivers-515")
         else:
             raise LisaException(
                 f"Distro {self._node.os.name}" "not supported to install CUDA driver."
```

Comment:
please update it 
`f"Distro {self._node.os.name} not supported to install CUDA driver."`